Sources/LocalReceiptParsing/Helpers/ReceiptParserLogger.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ReceiptParserLogger.swift
+//
+//  Created by Nacho Soto on 12/5/22.
+
+import Foundation
+
+final class ReceiptParserLogger: LoggerType {
```

Comment:
Hmm I believe this is currently unused?

Reply:
Hmm maybe, I split this PR in 2, so it’s possible this will be used by #2155